packages/typespec-ts/test/modularIntegration/generated/azure/core/lro/rpc/generated/src/api/operations.ts

```diff
@@ -22,11 +17,7 @@ export function _longRunningRpcSend(
   context: Client,
   body: GenerationOptions,
   options: LongRunningRpcOptionalParams = { requestOptions: {} },
-): StreamableMethod<
-  | LongRunningRpc202Response
-  | LongRunningRpcDefaultResponse
-  | LongRunningRpcLogicalResponse
-> {
+): StreamableMethod {
   return context
     .path("/azure/core/lro/rpc/generations:submit")
```

Comment:
pathUnchecked ?

Reply:
Good catch, but unfortunately using `pathUnchecked` doesn't work due to this bug I discovered while trying to apply your suggestion: https://github.com/Azure/azure-sdk-for-js/issues/30654. Using `path` directly bypasses the bug as by default `path` has type `Function`, which is less strict than `pathUnchecked`. I would like to proceed with using `path` for now and once we fix the Core bug we can revisit and change this to `pathUnchecked`.